static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-love-boat.js

```diff
@@ -0,0 +1,143 @@
+define([
+    'bean',
+    'common/modules/commercial/contributions-utilities',
+    'common/utils/template',
+    'common/utils/config',
+    'common/utils/$',
+    'common/views/svgs',
```

Comment:
Good point on the update! And on your second point - we probably don't need all the SVGs in the compiled file (regis and simon discussed this earlier today funnily enough) - it's about 30k or something and we probably don't use all of them, so importing them separately lends itself better to future treeshaking and reduction in size. I think Simon is going to convert them all soon.